charts/workflow/requirements.yaml

```diff
@@ -45,6 +45,7 @@ dependencies:
     version: <registry-token-refresher-tag>
     repository: https://charts.deis.com/registry-token-refresher
   - name: router
+    condition: !global.experimental_native_ingress
```

Comment:
No, we wouldn't include deis-router in that case. The debate is whether to use Helm 2.2.x's tags & conditions feature as the mechanism to disable deis-router, or whether to use our convention of big `{{ if }}` blocks which is more backward-compatible and consistent with other components.

I want to argue that we turn off deis-router "the old way," then make a separate effort to refactor all cases to use the new Helm functionality. But it may not be that simple; see https://github.com/deis/workflow/issues/771.